Does this module rely on d3-geo?

I do not see d3-geo in the dependencies but it seems to require d3-geo.
I found this issue when I use broswerify trying to create the map on the frontend.

This is the error message:

> Error: Can't walk dependency graph: Cannot find module 'd3-geo' from '/node_modules/geo-albers-usa-territories/dist/geo-albers-usa-territories.js'
>     required by /node_modules/geo-albers-usa-territories/dist/geo-albers-usa-territories.js

--- a/package.json
+++ b/package.json
@@ -31,6 +31,10 @@
   },
   "homepage": "https://github.com/stamen/geo-albers-usa-territories#readme",
   "devDependencies": {
-    "rollup": "^2.18.1"
+    "rollup": "^2.26.11",
+    "d3-geo": "^2.0.1"
+  },
+  "peerDependencies": {
+    "d3-geo": "^2.0.1"
   }
 }

